test/app-forms.spec.js

```diff
@@ -105,7 +107,16 @@ describe('forms that have caused bugs', () => {
     });
   });
 
-  it('d-tree infant child form attempts to fill questions in disabled section of form', async () => {
+  // Form replies heavily on default value of non-relevant values
+  // refer_neonatal_danger_sign_flag, 
+  /*
+  this express appears true even when child_consent_today is no?
+
+  not(selected( /infant_child/consent/child_consent_today ,"no")) and 
+  /infant_child/neonatal_danger_signs/refer_neonatal_danger_sign_flag  = 0 and 
+  ( /infant_child/age_days  >= 20 and  /infant_child/age_days  < ( /infant_child/week  * 15)) or ( /infant_child/age_days  >= ( /infant_child/month  * 6) and  /infant_child/age_days  < ( /infant_child/month  * 9)) or ( /infant_child/age_days  >= ( /infant_child/month  * 12) and  /infant_child/age_days  < ( /infant_child/month  * 15))
+  */
+  it.skip('d-tree infant child form attempts to fill questions in disabled section of form', async () => {
```

Comment:
test still failing? i can't remember why

Reply:
I think I got it sorted out!  The `dtree-infant-child` had a bunch of `relevant` expressions that were going off the rails because they were comparing non-relevant number values `= 0` (whereas now these values are `NaN`).  I pushed my changes to the form here. They are not elegant (just slapped `coalesce` in a bunch of places), but we should be able enable this test again now!.